apps/launchpad/app/components/app-sidebar.tsx

```diff
@@ -164,7 +164,7 @@ export function AppSidebar({
     <>
       <SidebarHeader className="px-5 py-3">
         <SidebarMenu className="gap-3">
-          <Link to="/dashboard" prefetch="intent">
+          <Link to="/dashboard">
```

Comment:
We may be able to add prefetching back in now that the prefetch calls are slimmer

Let's hold off since this works though ha